docs/manual/docs/cmd/todo/list/list-remove.md

```diff
@@ -0,0 +1,37 @@
+# todo list remove
+
+Removes a Microsoft To Do task list
+
+## Usage
+
+```sh
+todo list remove [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-n, --name [name]`|The name of the task list to remove. Specify either id or name but not both
+`-i, --id [id]`|The ID of the list to remove. Specify either id or name but not both
+`--confirm`|Don't prompt for confirming removing the task list
+`--query [query]`|JMESPath query string. See http://jmespath.org/ for more information and examples
+`-o, --output [output]`|Output type. `json,text`. Default `text`
+`--pretty`|Prettifies `json` output
+`--verbose`|Runs command with verbose logging
+`--debug`|Runs command with debug logging
+
```

Comment:
Let's add add a Remarks section that states that the API the command uses is in preview.

Reply:
Let me know if the remark I added is good for you :)

Reply:
Let's go with the below to be consistent with other commands, apologies, I should have suggested this as the update in my comment. This is what I updated the `add` command to use 👍🏻

```
## Remarks

!!! attention
    This command is based on an API that is currently in preview and is subject to change once the API reached general availability.
```

Reply:
Done :)